src/utils/amounts.ts

```diff
@@ -12,3 +20,7 @@ export const convertTokenAmountToFiat = ({ amount, token }: TokenAmountToFiatPar
   const fiat = amount * rates[token];
   return `$${fiat.toFixed(2)}`;
 };
+
+export const convertBigNumberToNumber = ({ amount, token }: BigNumberToNumberParams): number => {
```

Comment:
The name `convertBigNumberToNumber` suggests "give me a BigNumber and I'll give you a number" but instead it requires a BigNumber and a Token. I'd rename it in the line of `tokenAmountToNumber` or `tokenAmountToDecimalNumber`.